### DISCUSSION TITLE
In Person OpenDCS Workshop Ideas

### DISCUSSION THREAD

**Discussion by NathanHolcomb** [opendcs/opendcs#293] [Q&A]
Please use this thread for identifying any potential locations for the next OpenDCS Workshop. I believe we are fairly flexible on the date but something in the February/March 2024 time frame would keep us on a roughly biannual schedule.

Just to throw out some ideas from the 8/29/23 Virtual Workshop:

- Vendor hosted with a base cost for all attendees
- Hotel Venue near International Airport (reserve a block of rooms and we should get the conference room for free or significantly discounted).
- HEC or another Govt venue (This would require that we get a list of all non US attendees well ahead of time to ensure there are no access issues)

**Comment by MikeNeilson** [comment]
University ACM or ASCE  clubs may also be willing to help get us discounts for on campus venues and be interested in particpating and having access to working professionals in firlds the schools dont usually properly relate to each other.

**Comment by agilmore2** [comment]
As we discussed, I'm also looking into whether Precision would be
interested in hosting in Colorado at a non-federal facility. March in
Colorado can be chancy weather for snow storms though.

**Reply by NathanHolcomb** [reply]
Thanks Andrew! I don't think we are beholden to those particular months if Precisions does decide they would like to host.

**Reply by MikeNeilson** [reply]
And probably still worth contacting the College clubs, and CSU Sacramento had a USGS field office in their own building on campus.

I did event with the robotics club and Sac State and the cost for the clubs to host events on campus was drastically lower than for an general outside organization. I suspect it would be similar at other schools.

**Comment by NathanHolcomb** [comment]
Just wanted to check in to see if we had any updates on locations for an in-person Workshop. I'll send an email to the OpenDCS listserv to poll the general community but mainly wanted to check if Andrew had any feedback regarding Precision or if Mike would be up for hosting at HEC again.

**Reply by MikeNeilson** [reply]
While we certainly wouldn't mind,  I think our classroom is thoroughly booked through the window we have.
